components/StartScreenFooter.vue

```diff
@@ -1,6 +1,8 @@
 <template>
-  <div class="fixed bottom-0 w-screen mb-2 opacity-50">
-    <div class="flex mx-auto gap-x-4 w-max">
+  <div class="fixed bottom-0 w-screen space-y-2 opacity-50">
```

Comment:
```suggestion
  <div class="fixed bottom-2 w-screen space-y-2 opacity-50">
```
Raise div off the bottom.